timed/projects/tests/test_billing_type.py

```diff
@@ -3,21 +3,27 @@
 from rest_framework.status import HTTP_200_OK, HTTP_403_FORBIDDEN
 
 from timed.conftest import setup_customer_and_employment_status
-from timed.projects.factories import BillingTypeFactory
 
 
 @pytest.mark.parametrize(
-    "is_employed, is_customer_assignee, is_customer, status_code",
+    "is_employed, is_customer_assignee, is_customer, project__customer_visible, expected, status_code",
     [
-        (False, True, False, HTTP_403_FORBIDDEN),
-        (False, True, True, HTTP_403_FORBIDDEN),
-        (True, False, False, HTTP_200_OK),
-        (True, True, False, HTTP_200_OK),
-        (True, True, True, HTTP_200_OK),
+        (False, True, False, False, 0, HTTP_403_FORBIDDEN),
```

Comment:
This doesn't test the case if the user is external. The current code would likely HTTP/500 in that situation